test/test_repo_template/setup.py

```diff
@@ -3,7 +3,7 @@
 from setuptools import setup
 
 with open('asv_test_repo/build_time_env.py', 'w') as f:
-    f.write("env = {{}}\n".format(repr(dict(os.environ))))
+    f.write("env = {{}}\n".format(repr(dict(os.environ))))  # noqa see #1144
```

Comment:
Can you simply change the string for `f'env = {os.environ}'`. Or if it doesn't work just `env = {}`. This code doesn't make any sense, it simply writes `env = {}`, but I guess the intention was to have the envionment, so let's try first if that works.

Reply:
Thank you,Yes,ran it locally it just returns `env = {}`

Reply:
I see, can you try with `f'env = {os.environ}'` which makes more sense.